   ml_ops/sm-datazone_import/cross-account-setup.md

```diff
@@ -0,0 +1,59 @@
+### Cross Account Setup
```

Comment:
Check with Siamak - should we say "Cross" account or "Multi" account or does it matter?